frontend/src/views/PdfReportView.vue

```diff
@@ -60,7 +60,7 @@
                     </v-app-bar>
                     <v-card-text
                         :style="{ cursor: selectedCursor }"
-                        @mouseover="draggable = true"
+                        @mouseover="isDraggable = true"
```

Comment:
isDraggable finde ich komisch, da is als prefix bei einem boolean-wert ja suggeriert, dass es sich um einen getter handelt. Würde da draggable einfach lassen

Reply:
es gibt bereits eine komponente die draggable heißt, daher ist draggable hier nicht wirklich möglich, da es sonst zu überschneidungen mit der komponente kommt. Ich kanns draggableCard nennen

Reply:
dann nenn es draggableCard. Das is irritiert mich einfach :D